lib/util/index.js

```diff
@@ -1,10 +1,7 @@
 "use strict";
 
+let fs = require("fs");
 let path = require("path");
-let crypto = require("crypto");
-
-exports.abort = abort;
-exports.repr = repr;
```

Comment:
FWIW, I had intentionally moved exports to the top (yay hoisting) so you don't have to go searching for them in the middle of random implementation details. 🤷